sonar-plugin-api/src/main/java/org/sonar/api/web/Widget.java

```diff
@@ -21,6 +21,8 @@
 
 /**
  * @since 1.11
+ * @deprecated since 6.2, as dashboards have been removed
  */
+@Deprecated
 public interface Widget extends View {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)